include/Core/EmailLog.php

```diff
@@ -203,4 +218,13 @@ public function get_plugin_file() {
 	public function get_store_url() {
 		return self::STORE_URL;
 	}
+
+	/**
+	 * Shows the Email Log only to allowed User roles set in the Plugin's settings page.
+	 */
+	public function remove_email_log_menu_by_user_role() {
+		if ( ! \EmailLog\Util\can_current_user_email_log() ) {
```

Comment:
Let's change this function to `can_current_user_view_email_log`

Reply:
Fixed